### PR TITLE
RunCat: Add version 1.10

### PR DESCRIPTION
**Pull request by leni8ec**
A cute running cat animation on your windows taskbar (based on CPU load).

https://github.com/Kyome22/RunCat_for_windows

### CODE REVIEW

**Comment by rashil2000** [comment]
- Remove bin field
- Use architecture block

**Comment by rashil2000** [comment]
50MB isn't very large by 2021 standards. If the app is self sufficient, it is preferrable.

Also, the app does nothing for me (both builds). There are plenty of issues in the repo regarding this.

File: bucket/runcat.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "url": "https://github.com/Kyome22/RunCat_for_windows/releases/download/1.10/RunCat_x64.zip",
+    "hash": "6715019acf4e436193155028a44bbb918db511ecd5c01342d7d6e74146f58ba0",
+    "bin": "RunCat.exe",
+    "shortcuts": [
+        [
+            "RunCat.exe",
+            "RunCat"
+        ]
+    ],
+    "checkver": {
```
Comment (wenmin92) [inline bucket/runcat.json:15]: Preferred `"checkver": "github"`.

**Pull request merged**